internal/apm/dotnetwindows.go

```diff
@@ -0,0 +1,159 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package apm
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/newrelic/k8s-agents-operator/api/current"
+	"github.com/newrelic/k8s-agents-operator/internal/util"
+)
+
+const (
+	envDotnetFrameworkClrEnableProfiling     = "COR_ENABLE_PROFILING"
```

Comment:
Variable naming should be consistent -- either prefix everything with `windows` or don't use it at all. I'm really not sure it's necessary to have that prefix, since the filename makes pretty clear that it's the Windows-based version.